components/Common/AvatarGroup/index.tsx

```diff
@@ -0,0 +1,51 @@
+'use client';
```

Comment:
Why did you add `'use client';`? We're not using React Server-Side/Client-Side components yet. Adding this shouldn't be done at the moment.

Reply:
I'm adding that because if it's put on app dir it's will throw an error. I did this in anticipation of the future. I'm convinced that APP dir will replace page dir in future Nextjs updates. But if the website team doesn't think it's necessary, I'll remove it.

Reply:
> I'm adding that because if it's put on app dir it's will throw an error. I did this in anticipation of the future.

Don't make unnecessary changes. We don't know when we're going to migrate to `app` router, and we definitely want to tackle this into another time.

Every other component is not using "use client".

> But if the website team doesn't think it's necessary, I'll remove it.

So, I'm not Website team?

Reply:
> So, I'm not Website team? Since I asked you to remove it and you said "no" lol

😄 I mean yeah I will remove it but I just want to be sure you have my POV

Reply:
I have your PoV. Still, we want to streamline the work here. So any work related to "app" router, even if it's one line, we want to properly do research in the future.